Content.Server/GameObjects/Components/Atmos/GasCanisterComponent.cs

```diff
@@ -86,6 +151,230 @@ private void AnchorUpdate()
             {
                 DisconnectFromPort();
             }
+            UpdateAppearance();
+        }
+
+        #endregion
+
+        /// <summary>
+        /// Manages what happens when an actor interacts with an empty hand on the canister
+        /// </summary>
+        /// <param name="eventArgs"></param>
+        /// <returns>True if the interaction has succeded</returns>
+        bool IInteractHand.InteractHand(InteractHandEventArgs eventArgs)
```

Comment:
I just realized this but this should be using `IActivate`, not `IInteractHand`.

Reply:
question: why specifically `IActivate` rather than `IInteractHand` (in particular, is this a situation which could then have to be changed back depending on implementation of tank attachment?)

Reply:
Wait, no, that would be InteractUsing. And should probably be a separate component anyway. Will change to IActivate.